kiosk/kiosk_browser/__init__.py

```diff
@@ -32,12 +32,15 @@ def start(kiosk_url, settings_url, toggle_settings_key, fullscreen = True):
         mainWidget.resize(QSize(round(screen_size.width() / 2), round(screen_size.height() / 2)))
         mainWidget.show()
 
-    # Quit application when receiving SIGINT
-    def on_SIGINT(signum, frame):
+    # Quit application gracefully when receiving SIGINT or SIGTERM
+    # Tries to ensure profile / localStorage is flushed to disk
```

Comment:
```suggestion
    # This is important to trigger flushing of in-memory DOM storage to disk
```

I understand why your comment is timid, but long term I often don't find that helpful. I would either just state the purpose, or if there are specific limitations you are aware of, briefly mention them.